vsifile/io/base.py

```diff
@@ -11,21 +11,21 @@
 from diskcache import Cache
 
 from vsifile.logger import logger
-from vsifile.settings import vsi_settings
+from vsifile.settings import VSISettings
 
+vsi_settings = VSISettings()
+
+# TTL Block Cache (in-memory)
 block_cache: TTLCache = TTLCache(
     maxsize=vsi_settings.cache_blocks_maxsize,
     ttl=vsi_settings.cache_blocks_ttl,
 )
 
-
-def init_header_cache():
-    """Create or Connect to Header Cache layer."""
-    logger.debug(f"Using {vsi_settings.cache_directory} Cache directory")
-    return Cache(
-        directory=vsi_settings.cache_directory,
-        size_limit=vsi_settings.cache_headers_maxsize,
-    )
+# TTL Header Cache (in-disk)
+header_cache: Cache = Cache(
+    directory=vsi_settings.cache_directory,
+    size_limit=vsi_settings.cache_headers_maxsize,
+)
```

Comment:
TO BE REVIEWED

The diskcache layer is created on function import 👀 

ref: https://github.com/grantjenks/python-diskcache/issues/71